### ISSUE TITLE
Packet loss after Kez update

### ISSUE THREAD

**Issue by AxisKiku** [ValveSoftware/Dota2-Gameplay#22616]
### Description

After the new update, I'm consistently getting packet loss every game.

### Screenshots

![Untitled](https://github.com/user-attachments/assets/7d019531-cbc4-4efa-b789-fd9ddfed179c)

**Comment by takeiteasy85** [comment]
I have the same exact problem and i made a post in reddit https://www.reddit.com/r/DotA2/comments/1gmtiro/packet_loss_after_update/

**Comment by AxisKiku** [comment]
> I have the same exact problem and i made a post in reddit https://www.reddit.com/r/DotA2/comments/1gmtiro/packet_loss_after_update/

Yeah I saw your post earlier and we have the same situation. I hope Valve resolve this as soon as possible.

**Comment by takeiteasy85** [comment]
Despite having packet loss out, I don't feel any effect in the game.how about you? maybe the game net graph is faulty.

**Comment by AxisKiku** [comment]
> Despite having packet loss out, I don't feel any effect in the game.how about you? maybe the game net graph is faulty.

That was my assumption as well, maybe the network metric is bugged but my friends are not having this issue.
I tried using another ISP and there was still packet loss.

**Comment by takeiteasy85** [comment]
I live in italy where do you live? maybe it is a routing issue

**Comment by Ya0Ya0** [comment]
I have the same issue. Same thing happens in overthrow but not when spectating. I'm from the us btw

**Comment by HowlingLunatic** [comment]
I too suffer this issue. Have tried numerous ways(given dns,vpn different types of connection from WiFi to my phone hotspot and .....) to fix it, all have failed. it was fine before the Kez update. Ever since, haven`t been able to play a game without less than 60% loss on any given server.
idk i hope they look into it. 
pst: i`ve made an issue some months ago regarding a bug when u add both dota plus autumn and weather effect rain. ALL THIS TIME still no one has looked into it. starting to feel LORD GABEN no longer cares about dota anymore.

**Comment by HowlingLunatic** [comment]
> Despite having packet loss out, I don't feel any effect in the game.how about you? maybe the game net graph is faulty.

At first i had thought so too, however, as a Pudge player I`m no longer able to make hooks as i used to cause heroes just seam to teleport some units forward or even to sides.   :(

**Comment by koala-lava** [comment]
Constant micro-lags(packet losses). Server Russia. Tried 2 other online games it's all right there. Was alright from November 9 to 12.

**Comment by AxisKiku** [comment]
@bwvalve can you please look into this? Thank you.

**Comment by koala-lava** [comment]
Here's my match ID 8041181830, played just right now. I was playing Void. Connection is terrible. ISP says everything is ok.

**Comment by takeiteasy85** [comment]
this issue isnt fixed yet i have seen many dota streamers with the same problem

**Comment by AxisKiku** [comment]
7.37e and still having the same issue.

**Comment by N4chtwolf** [comment]
Same issue here, Since the Patch November 7th, Packet loss up to 10-80%, EU-West and EU-East,
Are your Player Reports/Behavior Score also not Working by Any Chance?

**Comment by vovannn271** [comment]
I'm having the same issue, 30-50 OUT packet loss. Only in dota. I can give you the match ID, im on venge: 8050280669

**Comment by myKiraa** [comment]
+ me too... 40-60% packet loss, but no effect lag? why? 

Metrics bug?

**Comment by guru-lakhima** [comment]
+1 / still the same / random huge packetloss + disconnecting numerous times in one game

**Comment by mintelectric22** [comment]
commenting for visibility, packet loss every game, while live on twitch it's 10 times worse. other games don't present any issue.. my ISP confirmed our internet is not the culprit.

**Comment by SFotakos** [comment]
Seems to still be happening, I get this randomly on US East and US West
![image](https://github.com/user-attachments/assets/0c1c9043-c0c9-4dc8-a33d-107eb77fd3da)

![image](https://github.com/user-attachments/assets/922bd880-1f7b-4041-9f43-28ec35a5ca95)
Dota is the only application affected

Edit:
Match info
![image](https://github.com/user-attachments/assets/8b6db665-ecfc-44d6-b8e2-d881b03caf58)

**Comment by Shaion7** [comment]
Also getting packet loss on US East server now, had 1-5% for the entire game, one time packet loss OUT jumped to 80% and i fed in lane because I could not move.

**Comment by AxisKiku** [comment]
Fixed on my end with the latest patch.
